fs_src/style.css

```diff
@@ -25,6 +25,12 @@ button {
   border-radius: 18px;
   border: none;
   cursor: pointer;
+  background: #0071e3;
+}
+
+button label {
+  position: relative;
+  cursor: pointer;

```

Comment:
Without this `cursor: pointer;` the mouse pointer was a hand on the button but a arrow on the button label